drivers/watchdog/CMakeLists.txt

```diff
@@ -50,6 +50,7 @@ zephyr_library_sources_ifdef(CONFIG_WDT_AMBIQ wdt_ambiq.c)
 zephyr_library_sources_ifdef(CONFIG_WDT_XMC4XXX wdt_xmc4xxx.c)
 zephyr_library_sources_ifdef(CONFIG_WWDT_NUMAKER wdt_wwdt_numaker.c)
 zephyr_library_sources_ifdef(CONFIG_WDT_ENE_KB1200 wdt_ene_kb1200.c)
+zephyr_library_sources_ifdef(CONFIG_WDT_IWDG_CH32V00X wdt_iwdg_ch32v00x.c)
```

Comment:
wdt_iwdg_ch32?

Reply:
Hmm, looking in drivers/ there are 6 with 'wch' in the name, 3 with ch32v00x in the name, and none with just ch32. I'll rename to `wdt_iwdg_wch.c` to match `wdt_iwdg_stm32.c`. Happy to revisit.

Reply:
Yea that is better, sorry.